topics/preprod-artifact-upload.yaml

```diff
@@ -0,0 +1,18 @@
+pipeline: generic-metrics
```

Comment:
I'm not sure what this value should be. Can this just be a new value (ex. `preprod-artifact` or something)?